### DISCUSSION TITLE
Created new WinForms .NET 8, project but controls still look like they're from Windows 98

### DISCUSSION THREAD

**Discussion by GuildOfCalamity** [dotnet/winforms#10339] [Q&A]
I was just watching the dotNET conf with Merrie where she demos the some of the new WinForms features; she mentions the updated controls, but when I create a new WinForms project using VS2022 Preview and .NET8 the controls still look outdated.
Is there a way to sample the new updated visual controls?
I would love to go back to WinForms one day (if the control style was updated). It's the main reason I went to UWP/WinUI3 was because of the outdated look.
You really don't want to deliver a product to your clients that looks like it was made using Windows 98.
[video](https://www.youtube.com/watch?v=N1weyWS_pL0)

**Comment by davidbuckleyni** [comment]
Only third party controls like devexpress or telerik offer themes. I had just asked a question for more material design
